data/modules/Scout/Scout.lua

```diff
@@ -365,8 +371,11 @@ local function calcSurfaceScanMission(sBody, difficulty, reward)
 		+ p.reward_interesting
 		* bodyReward
 
+	-- finally, convert the coverage into km2 and limit the number of decimals
+	local coverageKm2 = math.floor(coverage * 10000) / 10000
```

Comment:
Similarly, `utils.round(coverage, 0.0001)`.

Reply:
Ditto; fixed in 778e5a38e2cf9c784a046d38e6b55c47bfb3e5f6